pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/EndringIOpptjeningAuto.kt

```diff
@@ -0,0 +1,263 @@
+            )
+            {
+                includePhrase(Ufoeretrygd.MeldeFraOmEventuellInntektOverskrift)
+                includePhrase(Ufoeretrygd.MeldeFraOmEventuellInntekt)
+            }
+
+            includePhrase(
+                KombinereUfoeretrygdMedInntekt.MeldeFraOmEndringerIInntekten(
+                    avkortningsinformasjon = ufoeretrygdOrdinaer1.avkortningsinformasjon,
+                    ufoeregrad = ufoeregrad
+                )
+            )
+
+            ifNotNull(
+                fellesbarn1, saerkullsbarn1
+            ) { fellesbarn, saerkullsbarn ->
+                showIf(fellesbarn.notNull() or saerkullsbarn.notNull()) {
+                    includePhrase(
+                        BarnetilleggEndringIOpptjening(
+                            barnetilleggFellesbarn = fellesbarn,
+                            barnetilleggSaerkullsbarn = saerkullsbarn,
+                            grunnbeloep = grunnbeloep,
+                            sivilstand = sivilstand
+                        )
+                    )
+                }
+            }
+
+            includePhrase(
+                Gjenlevendetillegg.HarGjenlevendetillegg(
+                    forventetInntekt = ufoeretrygdOrdinaer1.avkortningsinformasjon.forventetInntekt,
+                    harGjenlevendetilleggInnvilget = harGjenlevendetilleggInnvilget.notNull(),
+                    inntektsgrense = ufoeretrygdOrdinaer1.avkortningsinformasjon.inntektsgrense,
+                )
+            )
+
+            includePhrase(
+                EndringIOpptjening.EtterbetalingAvUfoeretrygd(
+                    avkortningsinformasjon = ufoeretrygdOrdinaer1.avkortningsinformasjon,
+                    harOektUtbetaling = harOektUtbetaling,
+                    ufoeregrad = ufoeregrad,
+                    virkningsDato = virkningsDato
+
+                )
+            )
+            includePhrase(
+                EndringIOpptjening.TilbakekrevingAvUfoeretrygd(
+                    harOektUtbetaling = harOektUtbetaling
+                )
+            )
+            includePhrase(Ufoeretrygd.MeldeFraOmEndringer)
+            includePhrase(Ufoeretrygd.RettTilAAKlage)
+            includePhrase(Felles.RettTilInnsynPesys)
+            includePhrase(Ufoeretrygd.SjekkUtbetalingene)
+            includePhrase(Ufoeretrygd.Skattekort)
+
+            includePhrase(
+                Ufoeretrygd.SkattForDegSomBorIUtlandet(
+                    brukerBorInorge = this.brukerBorInorge
+                )
+            )
+        }
+
+
+        includeAttachment(vedleggMaanedligUfoeretrygdFoerSkatt, maanedligUfoeretrygdFoerSkatt)
+
+        includeAttachment(
+            createVedleggOpplysningerBruktIBeregningUT(
+                skalViseTrygdetidenDin = true,
+                skalViseMinsteytelse = true,
+                skalViseBarnetillegg = true,
+                skalViseAvdoed = true,
```

Comment:
Vi må se litt på disse styringene sammen for å se om vi kan få færre. Vi bør utfordre litt hvordan det har vært gjort tidligere i extream.